README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01 This is a line from RStudio
+
+##A Brief Biography of Myself, Wael Taji Miller
```

Comment:
Whoops! Make sure you include a space after your hashtags to mark a header. Otherwise the won't render! You run into this problem several times in this assignment.